Tdex-daemon unexpected crash after calling sell grpc function

With this request payload of `trade.sell` method of the TypeScript `tdex-sdk` module :
```
{
  "market": {
    "quoteAsset": "30973a9164ea895605161bb9f81c87938984620e102fa95f2564bd5ddf4ca5e1",
    "provider": "Tdex test",
    "feeBasisPoint": 25,
    "baseAsset": "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225"
  },
  "amount": 50000000,
  "asset": "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225",
  "identity": {
    "type": 4,
    "restorer": {
      "esploraEndpoint": "http://localhost:5001"
    },
    "network": {
      "messagePrefix": "\u0018Liquid Signed Message:\n",
      "bech32": "ert",
      "blech32": "el",
      "bip32": {
        "public": 70617039,
        "private": 70615956
      },
      "pubKeyHash": 235,
      "scriptHash": 75,
      "wif": 239,
      "confidentialPrefix": 4,
      "assetHash": "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225"
    },
    "isRestored": {
      "_h": 0,
      "_i": 1,
      "_j": true,
      "_k": null
    }
  }
}
```
The daemon crashes and after restarting it is still broken and I've had to delete and create the container again.
The version that I'm using is 0.8.1
These are the logs that i can see:
```
panic: runtime error: index out of range [0] with length 0

goroutine 4603 [running]:

github.com/tdex-network/tdex-daemon/pkg/wallet.extractScriptTypesFromPset(0x400043da40, 0x459d50, 0x40101f7f00, 0x10, 0x10, 0xcce920, 0xcce920, 0x14d6060, 0x0, 0x0, ...)

/tdex-daemon/pkg/wallet/util.go:192 +0xc14

github.com/tdex-network/tdex-daemon/pkg/wallet.calcFeeAmount(0x400043da40, 0x1, 0x1, 0x64, 0x1)

/tdex-daemon/pkg/wallet/util.go:252 +0x30

github.com/tdex-network/tdex-daemon/pkg/wallet.(*Wallet).UpdateTx(0x40000af560, 0x4010397a80, 0x198c, 0x4010287680, 0x3, 0x3, 0x0, 0x0, 0x0, 0x4010287bf0, ...)

/tdex-daemon/pkg/wallet/transaction.go:429 +0x8d4

github.com/tdex-network/tdex-daemon/internal/core/application.fillProposal(0x40006d4900, 0x18, 0x18, 0xfa8068, 0x400067d0e0, 0x4010320800, 0x4, 0x4, 0x4010287680, 0x3, ...)

/tdex-daemon/internal/core/application/trade_service.go:675 +0x3b8

github.com/tdex-network/tdex-daemon/internal/core/application.tradeManager.FillProposal(...)

/tdex-daemon/internal/core/application/types.go:377

github.com/tdex-network/tdex-daemon/internal/core/application.(*tradeService).TradePropose(0x4000090880, 0xfa0f20, 0x4010237d10, 0x4000686c40, 0x40, 0x4000686c80, 0x40, 0x1, 0xfa8068, 0x400067d0e0, ...)

/tdex-daemon/internal/core/application/trade_service.go:342 +0x8cc

github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.proposeTrade(0xfa3078, 0x4000090880, 0xfa0f20, 0x4010237d10, 0x401021f948, 0x419328, 0x400011f490, 0x10)

/tdex-daemon/internal/interfaces/grpc/handler/trade.go:221 +0x228

github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.tradePropose(0xfa3078, 0x4000090880, 0xfa70c8, 0x400011f490, 0x4000569400, 0xcc7920, 0xcba901)

/tdex-daemon/internal/interfaces/grpc/handler/trade.go:298 +0x94

github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.TradePropose(...)

/tdex-daemon/internal/interfaces/grpc/handler/trade.go:64

github.com/tdex-network/tdex-protobuf/generated/go/trade._Trade_TradePropose_Handler(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x40001f2040, 0xd2f540)

/go/pkg/mod/github.com/tdex-network/tdex-protobuf@v0.0.0-20211025103804-7ead2ca5a7b3/generated/go/trade/trade_grpc.pb.go:314 +0xf8

github.com/grpc-ecosystem/go-grpc-middleware.ChainStreamServer.func1.1(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x1, 0x50)

/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.0.1-0.20190118093823-f849b5445de4/chain.go:71 +0xe8

github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/interceptor.streamLogger(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x40006530e0, 0x4000116b90, 0x48, 0xffff98623f18)

/tdex-daemon/internal/interfaces/grpc/interceptor/logger.go:27 +0xa8

github.com/grpc-ecosystem/go-grpc-middleware.ChainStreamServer.func1.1(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x0, 0x0)

/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.0.1-0.20190118093823-f849b5445de4/chain.go:74 +0x90

github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/interceptor.streamMacaroonAuthHandler.func1(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x40006530e0, 0x4000116b90, 0x40106bb408, 0x9002bc)

/tdex-daemon/internal/interfaces/grpc/interceptor/auth.go:42 +0xb8

github.com/grpc-ecosystem/go-grpc-middleware.ChainStreamServer.func1(0xcf73a0, 0x400038ae10, 0xfa4a50, 0x40006a60c0, 0x40006530e0, 0xec5f58, 0xfa0f20, 0x4010237d10)

/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.0.1-0.20190118093823-f849b5445de4/chain.go:79 +0x138

google.golang.org/grpc.(*Server).processStreamingRPC(0x40003c6c40, 0xfa8a38, 0x401133e780, 0x40006547e0, 0x4000126b40, 0x143e2a0, 0x0, 0x0, 0x0)

/go/pkg/mod/google.golang.org/grpc@v1.40.0/server.go:1557 +0x574

google.golang.org/grpc.(*Server).handleStream(0x40003c6c40, 0xfa8a38, 0x401133e780, 0x40006547e0, 0x0)

/go/pkg/mod/google.golang.org/grpc@v1.40.0/server.go:1630 +0xa28

google.golang.org/grpc.(*Server).serveStreams.func1.2(0x400043b280, 0x40003c6c40, 0xfa8a38, 0x401133e780, 0x40006547e0)

/go/pkg/mod/google.golang.org/grpc@v1.40.0/server.go:941 +0x94

created by google.golang.org/grpc.(*Server).serveStreams.func1

/go/pkg/mod/google.golang.org/grpc@v1.40.0/server.go:939 +0x1f8
```

--- a/pkg/wallet/util.go
+++ b/pkg/wallet/util.go
@@ -189,7 +189,7 @@ func extractScriptTypesFromPset(ptx *pset.Pset) ([]int, []int, []int, []int, []i
 				scriptSize := 1 + (1+72)*m + 1 + varIntSerializeSize(uint64(scriptLen)) + scriptLen
 				inAuxiliaryWitnessSize = append(inAuxiliaryWitnessSize, scriptSize)
 			} else if in.RedeemScript != nil {
-				inScriptTypes[i] = P2SH_P2WPKH
+				inScriptTypes = append(inScriptTypes, P2SH_P2WPKH)
 			}
 		case address.P2WpkhScript:
 			inScriptTypes = append(inScriptTypes, P2WPKH)

